### PR TITLE
Maximremizov ui development

### PR DESCRIPTION
**Pull request by remizovMaxim**

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_DB_CONNECT=
```
Comment (DarkProDen) [inline .env.template:1]: Not full template

File: pages/login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </Head>
+        <Header>Contact app</Header>
+        <Wrapper>
+            <Logo />
+            <BlockHeader>Войти</BlockHeader>
+            <form>
+                <FormInput placeholder='Username'></FormInput>
+                <FormInput placeholder='Password'></FormInput>
+                <FormLink>Нет учётной записи? <a href='/registration'>Создайте её.</a></FormLink>
+                <FormLink><a href='#'>Не удаётся получить доступ к своей учетной записи?</a></FormLink>
```
Comment (DarkProDen) [inline pages/login/index.tsx]: Why # ?

File: components/AsideBlock/AsideBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { AsideBlockWrapper } from "./style"
+
+const AsideBlock = () => {
+    return (<AsideBlockWrapper>
+
+    </AsideBlockWrapper>)
```
Comment (DarkProDen) [inline components/AsideBlock/AsideBlock.tsx]: should use a single tag

File: components/AvatarBlock/AvatarBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { Wrapper, Avatar } from "./style";
+
+const AvatarBlock = () => {
+    return (<Wrapper>
+        <Avatar src='/default_avatar.png' alt='avatar'/>
+    </Wrapper>)
```
Comment (DarkProDen) [inline components/AvatarBlock/AvatarBlock.tsx]: code style

File: components/Button/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { ButtonBlue, ButtonGrey } from './style';
+
+export const typeButton = {
+    next: "next",
+    prev: "prev"
+}
+
+type PropsType = {
+    type: typeof typeButton.next | typeof typeButton.prev
```
Comment (DarkProDen) [inline components/Button/Button.tsx]: what is this?

File: pages/login/index.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                <ButtonsBlock>
+                    <Button name='Назад' type={typeButton.prev} />
+                    <Button name='Далее' type={typeButton.next} />
```
Comment (DarkProDen) [inline pages/login/index.tsx]: don't use "type" like a custom prop

File: .env.template (hunk 1 above)
Comment (remizovMaxim) [reply .env.template:1]: Each developer must enter the data to run locally.

File: pages/login/index.tsx (hunk 2 above)
Comment (remizovMaxim) [reply pages/login/index.tsx]: because I don’t know where this link should lead

File: components/AsideBlock/AsideBlock.tsx (hunk 3 above)
Comment (remizovMaxim) [reply components/AsideBlock/AsideBlock.tsx]: soon there will be a new commit where I will add content to these tags

**Pull request closed without merging**